lib/posts.js

```diff
@@ -220,4 +221,48 @@ PostsRequest.prototype.statuses = function() {
 	return this;
 };
 
+/**
+ * @method year
+ * @chainable
+ * @param year - Integer for year
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.year = function ( year ) {
+  return this.filter( 'year', year );
+};
+
+/**
+ * @method month
+ * @chainable
+ * @param month - Integer for month or month string ("january")
+ * @returns {CollectionRequest} The PostsRequest instance (for chaining)
+ */
+PostsRequest.prototype.month = function ( month ) {
+  if ( _.isString( month ) ) {
+    month = new Date( Date.parse(month +" 1, 2012") );
```

Comment:
I understand what is being done here, but I'd recommend leaving a comment to explain what's being accomplished with this apparently "magical" string.